priv/repo/migrations/20220722074206_create_keyword_urls.exs

```diff
@@ -0,0 +1,17 @@
+defmodule GoogleScraping.Repo.Migrations.CreateKeywordUrls do
+  use Ecto.Migration
+
+  def change do
+    create table(:keyword_urls) do
+      add :url, :text, null: false
+      add :is_ad, :boolean
```

Comment:
How about setting a default value and null false to avoid  three state boolean problem https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem

Reply:
Good catch. For some reason I thought that ORM will make boolean field not null by default.

The fix is here: https://github.com/Nihisil/nimble-certification-elixir/pull/52/commits/cfccb6db7caf78005e9619fec82f658cd9bdd639